site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -9,6 +9,80 @@ docs: "DOCS-1419"
 
 This topic describes possible issues users might encounter when using NGINX Gateway Fabric. When possible, suggested workarounds are provided.
 
+### General Troubleshooting
+
+When attempting to diagnose a problem or get support, there are a few important data points that can be collected to help with understanding what issues may exist.
+
+##### Resource Status
+
+To get the status of a resource, use `kubectl describe`. For example, to check the status of the `coffee` HTTPRoute, which has an error:
+
+```shell
+kubectl describe httproutes.gateway.networking.k8s.io coffee [-n namespace]
+```
+
+```text
+...
+Status:
+  Parents:
+    Conditions:
+      Last Transition Time:  2024-05-31T17:20:51Z
+      Message:               The route is accepted
+      Observed Generation:   4
+      Reason:                Accepted
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2024-05-31T17:20:51Z
+      Message:               spec.rules[0].backendRefs[0].name: Not found: "bad-backend"
+      Observed Generation:   4
+      Reason:                BackendNotFound
+      Status:                False
+      Type:                  ResolvedRefs
+    Controller Name:         gateway.nginx.org/nginx-gateway-controller
+    Parent Ref:
+      Group:         gateway.networking.k8s.io
+      Kind:          Gateway
+      Name:          gateway
+      Namespace:     default
+      Section Name:  http
+```
+
+If a resource has any errors relating to its configuration or relation to other resources, it is likely that those errors will be contained within the status. The `ObservedGeneration` in the status should match the `ObservedGeneration` of the resource. Otherwise, this could mean that the resource wasn't processed yet or the status failed to update.
```

Comment:
```suggestion
If a resource has errors relating to its configuration or relationship to other resources, they can likely be read in the status. The `ObservedGeneration` in the status should match the `ObservedGeneration` of the resource. Otherwise, this could mean that the resource hasn't been processed yet or that the status failed to update.
```
This is mostly tense consistency.